src/api/query-hooks/mutations/useUpdateEvidenceDOD.tsx

```diff
@@ -0,0 +1,38 @@
+import { useMutation } from "@tanstack/react-query";
+import { Evidence, updateEvidence } from "../../services/evidence";
+
+export default function useUpdateEvidenceDOD(param?: {
```

Comment:
Why do we need a specific mutation for the definition of done? Why not just updateEvidence ?